src/lib/Server/Output/ValueObjectVisitor/BookmarkList.php

```diff
@@ -18,7 +18,7 @@ class BookmarkList extends ValueObjectVisitor
     /**
      * {@inheritdoc}
```

Comment:
Following the similar case before:
```suggestion
     * @param \Ibexa\Rest\Server\Values\BookmarkList $data
```